### PR TITLE
feat: support disable prompt lowercase

### PR DESCRIPTION
**Pull request by lgc2333**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -164,6 +165,7 @@ export const PromptConfig: Schema<PromptConfig> = Schema.object({
   translator: Schema.boolean().description('是否启用自动翻译。').default(true),
   latinOnly: Schema.computed(Schema.boolean(), options).description('是否只接受英文输入。').default(false),
   maxWords: Schema.computed(Schema.natural(), options).description('允许的最大单词数量。').default(0),
+  lowercase: Schema.boolean().description('是否将输入的标签转换为小写。').default(true),
```
Comment (MaikoTan) [inline src/config.ts:168]: You could put this line before `maxWords`.

**Review by MaikoTan — APPROVED** [review]

File: src/config.ts [hunk 2]
```diff
@@ -148,6 +148,7 @@ export interface PromptConfig {
   placement?: Computed<'before' | 'after'>
   latinOnly?: Computed<boolean>
   translator?: boolean
+  lowercase?: boolean
```
Comment (shigma) [inline src/config.ts:151]:
```suggestion
  lowerCase?: boolean
```

File: src/config.ts [hunk 3]
```diff
@@ -163,6 +164,7 @@ export const PromptConfig: Schema<PromptConfig> = Schema.object({
   ]), options).description('默认附加标签的位置。').default('after'),
   translator: Schema.boolean().description('是否启用自动翻译。').default(true),
   latinOnly: Schema.computed(Schema.boolean(), options).description('是否只接受英文输入。').default(false),
+  lowercase: Schema.boolean().description('是否将输入的标签转换为小写。').default(true),
```
Comment (shigma) [inline src/config.ts:167]:
```suggestion
  lowerCase: Schema.boolean().description('是否将输入的标签转换为小写。').default(true),
```

File: src/config.ts [hunk 4]
```diff
@@ -422,7 +424,8 @@ export function parseInput(session: Session, input: string, config: Config, over
     const tags = input.split(/,\s*/g)
     if (placement === 'before') tags.reverse()
     for (let tag of tags) {
-      tag = tag.trim().toLowerCase()
+      tag = tag.trim()
+      if (config.lowercase) tag = tag.toLowerCase()
```
Comment (shigma) [inline src/config.ts:428]:
```suggestion
      if (config.lowerCase) tag = tag.toLowerCase()
```

**Review by shigma — CHANGES_REQUESTED** [review]
写 python 写的

**Review by MaikoTan — APPROVED** [review]

**Pull request merged**